DeusEx/Classes/DXRMenuScreenNewGame.uc

```diff
@@ -20,11 +20,15 @@ function SetDxr(DXRando d)
 
 function CopySkills()
 {
+    local DXRSkills dxrs;
     local Skill aSkill;
     local int skillIndex;
     
-    if(dxr!=None)
-        DXRSkills(dxr.LoadModule(class'DXRSkills')).RandoSkills();
+    if(dxr!=None) {
+        dxrs = DXRSkills(dxr.FindModule(class'DXRSkills'));
+        if( dxrs != None )

```

Comment:
if DXRSkills isn't found, then the user must've removed it from the modules_to_load